clang/test/CIR/CodeGenOpenACC/openacc-not-implemented.cpp

```diff
@@ -0,0 +1,20 @@
+// RUN: %clang_cc1 -std=c++17 -triple x86_64-unknown-linux-gnu -fopenacc -fclangir -emit-cir %s -o %t.cir -verify
+// RUN: %clang_cc1 -std=c++17 -triple x86_64-unknown-linux-gnu -fopenacc -fclangir -emit-llvm %s -o %t-cir.ll -verify
+
+void HelloWorld(int *A, int *B, int *C, int N) {
+
+  // expected-error@+2{{ClangIR code gen Not Yet Implemented: OpenACC Compute Construct}}
+  // expected-error@+1{{ClangIR code gen Not Yet Implemented: statement}}
```

Comment:
super nit: if these are to be aligned with the content after the pragma, might look better with @-1 and @-2 (or alternatively be indented with the `#` below?

Reply:
*sigh*... yeah, `#pragma` ends up getting indented automatically to be left-aligned, not scope-aligned, which is a touch annoying (Clang-format AND vim do it).  So I've moved the comments to the 'left' so at least they are aligned.